movie-showcase/src/routes/display/+page.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+    import Carousel from 'svelte-carousel'	
+    import { writableArray } from '../../stores';
+    import { slideDuration } from '../../stores';
+    import { carouselItemCount } from '../../stores';    
+    import { onMount } from "svelte";
+
+    onMount(checkSettings);
+
+    function checkSettings(){
+        if($slideDuration == 0 || $slideDuration == null) $slideDuration = 5;
+        if($carouselItemCount == 0 || $carouselItemCount == null) $carouselItemCount = 1;
+    }
+</script>
+
+{#if $writableArray.length >= 1} 
```

Comment:
`writableArray` as a name is not very descriptive, when reviewing the code (or maintaining it in the future) it's impossible to know what it actually contains without looking at the definition